api/db/changelog/db.changelog-10.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="calbach" id="changelog-10">
+    <addColumn tableName="workspace">
+      <column name="version" type="smallint">
+        <constraints nullable="false"/>
+      </column>
+    </addColumn>
+    <addColumn tableName="cohort">
```

Comment:
We don't have to do it in this PR necessarily, but I think we want one of these on user, too.